src/modules/AuthenticationModule/components/Verify Account/VerifyAccount.tsx

```diff
@@ -1,8 +1,103 @@
 
+import { useContext } from "react";
+import logo from "../../../../assets/PMS 3.png";
+import { APIcontext } from "../../../../Context/baseUrl";
+import { useForm } from "react-hook-form";
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
+import { ToastContext } from "../../../../Context/ToastContext";
+
 const VerifyAccount = () => {
+  const {baseUrl}=useContext(APIcontext)
+  const {getToastValue}=useContext(ToastContext)
+
+  const navigate = useNavigate();
+
+  const {
+      register,
+      handleSubmit,
+      formState: { errors },
+  } = useForm();
+  const onSubmit = async (data) => {
+    try {
+        let response = await axios.put(
+            `${baseUrl}/Users/verify`,data
+        );
+        getToastValue('success','Verification success you can login now')
+        navigate("/login");
+    } catch (error) {
+      getToastValue('error',error.response.data.message)
+    }
+};
   return (
     <>
-      VerifyAccount
+    <div className="register-container">
+                <div className="container-fluid vh-100">
+                    <div className="row vh-100 justify-content-center align-items-center">
+                        <div className="col-md-6">
+                            <div className="text-center">
+                                <img src={logo} alt="" />
+                            </div>
+                            <div className="register-form rounded-5 mt-4 p-5">
+                                <div className="ms-4">
+                                    <p className="text-light m-0">welcome to PMS</p>
+                                    <h3 className="register-text-color ">
+                                        <span className="underLine">V</span>erify Account
+                                    </h3>
+                                </div>
+                                <form onSubmit={handleSubmit(onSubmit)} className=" pt-4 pb-5">
+                                    <div className="row justify-content-evenly">
+                                        <div className="col-md-11">
+                                            <div className="input-group pb-3">
+                                                <p className="my-1 register-text-color">E-mail</p>
+                                                <input
+                                                    type="text"
+                                                    className="input-style"
+                                                    placeholder="Enter your E-mail"
+                                                    {...register("email", {
+                                                        required: "Email is required",
```

Comment:
Email, password and other validation will be used in multiple places. I consider making a file containing all needed validations and then use this validation in our application. This will make our code more concise.

You can look at this example:
https://github.com/AnasKhaled7/PMS-React/blob/main/src/lib/InputValidator.ts

Reply:
how can i store this in inputValidator.ts 'watch , value '?
'{...register("confirmPassword", {
                                                        required: "Confirm Password is required",
                                                        validate: (value) =>
                                                            value === watch("password") ||
                                                            "Passwords do not match",
                                                    })}'